wcivf/apps/elections/views/postcode_view.py

```diff
@@ -86,9 +86,9 @@ def get(self, request, *args, **kwargs):
             event["summary"] = "{} - {}".format(
                 post_election.election.name, post_election.post.label
             )
-
-            event.add("dtstart", post_election.election.start_time)
-            event.add("dtend", post_election.election.end_time)
+            event.add("dtstamp", timezone.now()),
```

Comment:
Hard to tell from the diff, but are the trailing `,` meant to be there?

It's valid python to do this, because you can make a tuple inline, e.g `x, y = 1, 2` (I don't think this is great python FYI, but it is valid).

If it's not inside a function call, I think they should be removed.

Reply:
I don't think they are meant to be there; will remove.